WooCommerce/WooCommerceTests/POS/ViewModels/PointOfSaleDashboardViewModelTests.swift

```diff
@@ -33,6 +33,32 @@ final class PointOfSaleDashboardViewModelTests: XCTestCase {
         super.tearDown()
     }
 
+    func test_plain_setup() {
+        XCTAssertEqual(sut.orderStage, .building)
+        XCTAssertEqual(sut.isCartCollapsed, true)
+        XCTAssertEqual(sut.isAddMoreDisabled, false)
+        XCTAssertEqual(sut.isExitPOSDisabled, false)
+    }
```

Comment:
I'd say let's use the naming pattern we have for other tests via Given/When/Then, so it's more readable to understand what the test is doing without having to check the assertion details. What do you think? This also helps to see which bits are not straight-away testable. Eg:

```suggestion
    func test_viewmodel_when_loaded_then_has_expected_initial_setup() {
        // Given
        let expectedCartCollapsedState = false
        let expectedAddMoreButtonDisabledState = false
        let expectedExitPOSButtonDisabledState = false

        // When/Then
        XCTAssertEqual(sut.isCartCollapsed, expectedCartCollapsedState)
        XCTAssertEqual(sut.isAddMoreDisabled, expectedAddMoreButtonDisabledState)
        XCTAssertEqual(sut.isExitPOSDisabled, expectedExitPOSButtonDisabledState)
    }
```

While we can assert the state of `sut.orderStage, .building` we cannot declare the type since `OrderStage` it's private, so perhaps we either make this internal/public if needed, or create a separate test just for the property for visibility.

We can do the same for the other two tests as well.